docs/tutorials/dockerimages.md

```diff
@@ -294,10 +293,7 @@ Now let's take our `Dockerfile` and use the `docker build` command to build an i
     Sending build context to Docker daemon
     Step 1 : FROM ubuntu:14.04
      ---> e54ca5efa2e9
-    Step 2 : MAINTAINER Kate Smith <ksmith@example.com>
-     ---> Using cache
-     ---> 851baf55332b
-    Step 3 : RUN apt-get update && apt-get install -y ruby ruby-dev
+    Step 2 : RUN apt-get update && apt-get install -y ruby ruby-dev
```

Comment:
Don't the following step numbers need to be updated too? (see `Step 4` following this now `Step 2`)